apps/explorer/src/components/module/module-functions-interaction/FuncitonParamInput.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { FieldPath, FieldValues, UseFormRegister } from 'react-hook-form';
+
+type FunctionParamInputProps<T extends FieldValues, N extends FieldPath<T>> = {
+    paramTypeTxt: string;
+    paramIndex: number;
+    register: UseFormRegister<T>;
+    id: string;
```

Comment:
Dump the ID prop, either use `useId`, or wrap the label with the input to get the correct a11y handling.